src/snowflake/cli/api/project/util.py

```diff
@@ -183,6 +195,34 @@ def get_env_username() -> Optional[str]:
     return first_set_env("USER", "USERNAME", "LOGNAME")
 
 
+DEFAULT_USERNAME = "unknown_user"
+
+
+def get_sanitized_username():
+    return clean_identifier(get_env_username() or DEFAULT_USERNAME)
+
+
+def concat_identifiers(identifiers: list[str]) -> str:
```

Comment:
nit: why not * identifiers so we can use the variadic version in internal logic too?

Reply:
does variadic version have any advantages? I thought this looks cleaner for use internally.

Reply:
subjective, let's leave as is